test-infra/npm-shrinkwrap.json

```diff
@@ -2,19 +2,19 @@
   "name": "bootstrap",
   "version": "4.0.0-alpha",
   "npm-shrinkwrap-version": "200.4.0",
-  "node-version": "v0.12.7",
+  "node-version": "v4.1.0",
```

Comment:
This was made automatically with `update-shrinkwrap` because I use 4.1.0 locally. I'm in the process of testing node.js 4 for Travis so we can merge that before this.